backend/test_parser.py

```diff
@@ -1,8 +1,36 @@
-from parser import Parser, Sample
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from parser import Sample
+from parser import Parser
 import unittest
 
+regex = {}
```

Comment:
These regex should be part of a different config somewhere. Then maybe your test, or whatever runs the program specifies the format type. We could call this "google_format" or something similar.